client/src/components/Header/Header.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
+import { FaApple } from "react-icons/fa";
+import { AiOutlineSearch } from "react-icons/ai";
+import { IoBagOutline } from "react-icons/io5";
+import "../../styles/Header/Header.scss";
+function Header() {
+  useEffect(() => {
+    const navIcon = document.querySelector(".nav-icon");
+    const nav = document.querySelector("nav");
+    navIcon.onclick = function () {
+      nav.classList.toggle("show");
+    };
+  }, []);
```

Comment:
Let's not use the real DOM please! It's very bad in here. 😞

Reply:
I'll try finding a workaround for it then😅